extensions/amp-slike/0.1/test/validator-amp-slike.out

```diff
@@ -0,0 +1,21 @@
+PASS
```

Comment:
Out file doesn't match the input file. You can regenerate the out file by running from the `validator/` dir

```
python build.py --update_tests
```